Core/include/Acts/Propagator/MaterialInteractor.hpp

```diff
@@ -94,8 +94,8 @@ struct MaterialInteractor {
 
       if (energyLoss) {
         using namespace UnitLiterals;
-        ACTS_VERBOSE(d.slab << " pdg=" << d.pdg << " mass=" << d.mass / 1_MeV
-                            << "MeV"
+        ACTS_VERBOSE(d.slab << " absPdg=" << d.absPdg
+                            << " mass=" << d.mass / 1_MeV << "MeV"
```

Comment:
Do we have the charge sign here? We could present a signed PDG just for information purposes.

Reply:
we could get the charge from the stepper. would it be as easy as `absPdg * charge * -1`?

Reply:
I suppose so?